src/cli/hook-utils/manifest.spec.js

```diff
@@ -76,50 +77,4 @@ describe('Slack CLI Script Hooks: get-manifest utilities', () => {
       expect(result).to.equal(null);
     });
   });
-  describe('manifest imports', () => {
```

Comment:
Unfortunately, the sinon.stub behavior wasn't doing what I initially thought it was doing but was actually failing silently :womp:

I've removed these test cases for the time being as I don't want to mislead. The utility functions that are as part of`readManifestJSONFile()` and `readImportedManifestFile()` are tested.

Reply:
Thanks. This makes sense and it's a good decision for future maintainers